tools/spi-hdlc-adapter/spi-hdlc-adapter.c

```diff
@@ -1632,6 +1636,18 @@ int main(int argc, char *argv[])
                 syslog(LOG_NOTICE, "SPI CS Delay set to %d usec", sSpiCsDelay);
                 break;
 
+            case ARG_SPI_RESET_DELAY:
+                assert(optarg);
+
+                sSpiResetDelay = atoi(optarg);
+                if (sSpiResetDelay < 0)
+                {
+                    syslog(LOG_ERR, "Negative values (%d) for --spi-reset-delay are invalid.", sSpiResetDelay);
```

Comment:
Negative value (%d) for --spi-reset-delay us invalid.

Reply:
Updated.